scipy/optimize/optimize.py

```diff
@@ -761,7 +737,20 @@ def approx_fprime(xk, f, epsilon, *args):
     array([   2.        ,  400.00004198])
 
     """
-    return _approx_fprime_helper(xk, f, epsilon, args=args)
+    xk = np.asarray(xk, float)
+
+    rel_step = _compute_relative_step(epsilon, xk, '2-point')
+    f0 = f(*((xk,) + args))
```

Comment:
Now that python 3.5 is the minimum version, I think you can use generalised argument unpacking
```suggestion
    f0 = f(xk, *args)
```